### PR TITLE
feature(unlock-app): Event Collection detail page

### PR DESCRIPTION
**Pull request by 0xTxbi**
# Description
This PR introduces a page to display an event collection's detail.

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/components/content/events-collection/EventDetailDawer.tsx [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+import { EventLocation } from '../event/EventLocation'
+
+interface EventDetailDrawerProps {
+  isOpen: boolean
+  setIsOpen: (isOpen: boolean) => void
+  event: Event | null
+}
+
+// Helper function to get ordinal suffix for a given day
+const getOrdinalSuffix = (day: number): string => {
```
Comment (julien51) [inline unlock-app/src/components/content/events-collection/EventDetailDawer.tsx:20]: There must be a library that does this, no? We already use `dayjs` and I suspect it support this!

File: unlock-app/src/components/content/events-collection/EventDetailDawer.tsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+      return 'nd'
+    case 3:
+      return 'rd'
+    default:
+      return 'th'
+  }
+}
+
+// util to format date range
+const formatDateRange = (startDateStr: string, endDateStr?: string): string => {
```
Comment (julien51) [inline unlock-app/src/components/content/events-collection/EventDetailDawer.tsx:35]: same!

File: unlock-app/src/components/content/events-collection/EventOverviewCard.tsx [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    return (
+      <div className="flex bg-white border border-gray-200 rounded-2xl p-4">
+        <p className="text-red-500">Event data is unavailable.</p>
+      </div>
+    )
+  }
+
+  const { image, ticket, attributes } = data
+
+  const getAttribute = (type: string) => {
```
Comment (julien51) [inline unlock-app/src/components/content/events-collection/EventOverviewCard.tsx:40]: Let's maybe use a mor eexplicit name like `getEventAttribute` ?

**Review by julien51 — CHANGES_REQUESTED** [review]
A few nits. Also please add screenshots i fyou can!

**Comment by 0xTxbi** [comment]
all changes made!

this PR contains element that only facilitate the initial configured state of a newly created collection.
event-collections-configured-state




PR #14651 upon successful CI tests, will contain screengrabs of the detail page with the existence of added events.

File: unlock-app/src/components/content/events-collection/icons/FarcasterIcon.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+interface FarcasterIconProps {
+  size?: number
+  className?: string
+}
+
+export default function FarcasterIcon({
```
Comment (julien51) [inline unlock-app/src/components/content/events-collection/icons/FarcasterIcon.tsx:6]: Let's use https://react-icons.github.io/react-icons/search/#q=farcaster

**Comment by 0xTxbi** [comment]
> Please please, let's not add own own icons, or own libraries to format dates

got it!

**Review by julien51 — APPROVED** [review]

**Pull request merged**